modules/storages/spec/features/storages/admin/project_storages_spec.rb

```diff
@@ -315,13 +329,23 @@
         expect(page).to have_text(project.name)
         project_storages_index_page.click_menu_item_of("Remove project", project)
 
+        # The original DeleteService would try to remove actual files from actual storages,
+        # which is of course not possible in this test since no real storage is used.
```

Comment:
🟢 Makes sense- good that the service is already well unit tested so we can confidently stub it out 👍🏾